platform/admin/ui/src/main/webapp/js/models/Service.js

```diff
@@ -212,6 +215,14 @@ define(['backbone', 'jquery','backboneassociations'],function (Backbone, $) {
                     return defaults;
                 }, {}));
             return this;
+        },
+        getConfigurationDisplayName: function () {
+            var displayName = this.get('id');
+            var properties = this.get('properties');
+            if (isServiceFactory(properties)) {
+                displayName = properties.get('name') || properties.get('shortname') || properties.get('id') || displayName;
```

Comment:
I assume `model.id` is different from `properties.get('id')`? If not, we should be able to remove `|| properties.get('id')` from the chain.